### PR TITLE
Update addresses to checksum

### PR DESCRIPTION
**Pull request by dudesahn**

### CODE REVIEW

File: icons/aliases.json [hunk 1]
```diff
@@ -250,30 +250,20 @@
...
-    "address": "0x19db27D2E9E4B780CF9A296D575aBbddEe1578DA"
+    "address": "0xe11ba472F74869176652C35D30dB89854b5ae84D"
   },
   {
-    "name": "HEGIC",
+    "name": "Hegic",
     "symbol": "HEGIC",
     "address": "0x584bC13c7D411c00c01A62e8019472dE68768430"
   },
-  {
```
Comment (milkyklim) [inline icons/aliases.json:267]: Can you put this back?

File: icons/aliases.json [hunk 2]
```diff
@@ -225,7 +225,7 @@
     "address": "0x29E240CFD7946BA20895a7a02eDb25C210f9f324"
   },
   {
-    "name": "aLINK",
+    "name": "Aave Interest bearing LINK",
```
Comment (milkyklim) [inline icons/aliases.json:228]: **B**earning

Comment (dudesahn) [reply icons/aliases.json:228]: it was lower-case "b" on Etherscan 😅

File: icons/aliases.json (hunk 1 above)
Comment (dudesahn) [reply icons/aliases.json:267]: Sure, it's not a live vault but I can add it

File: icons/aliases.json (hunk 2 above)
Comment (dudesahn) [reply icons/aliases.json:228]: Done

**Pull request merged**